terraform/environments/data-platform/iam.tf

```diff
@@ -18,20 +18,23 @@ data "aws_iam_policy_document" "log_to_bucket" {
 
 data "aws_iam_policy_document" "read_metadata" {
   statement {
-    sid    = "s3ReadMetadata"
-    effect = "Allow"
-    actions = [
-      "s3:GetObject",
-      "s3:ListBucket",
-    ]
-    resources = [
-      "${module.metadata_s3_bucket.bucket.arn}",
-      "${module.metadata_s3_bucket.bucket.arn}/*"
-    ]
+    sid       = "s3ReadMetadata"
```

Comment:
I think i added list bucket in here because the create schema lambda uses `list_objects_v2` which requires the `ListBucket` permission. It could go somewhere else as don't think it's used by create metadata lambda, but yeah probably not the most suitable place for it